emqx/manifest.json

```
@@ -0,0 +1,57 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fa40ec7e-e8f6-4c4b-a675-31716b23a9df",
+  "app_id": "emqx",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect openmetrics v2 data from EMQX.",
+    "title": "EMQX",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "EMQX Broker Metrics in Datadog dashboard",
+        "image_url": "images/emqx-dashboard.png"
+      }
+    ]
```

```suggestion
    ],
```This may fix your invalid JSON error